core/src/test/resources/fixtures/issue69/policy.rego

```diff
@@ -0,0 +1,9 @@
+package authz
+
+import rego.v1
```

Comment:
I got the error:
```
rego_parse_error: var cannot be used for rule name
```

and solved following these instructions:
https://docs.styra.com/opa/errors/rego-parse-error/var-cannot-be-used-for-rule-name

Reply:
Which OPA version are you using? That import isn't needed since 1.0+, and shouldn't be used for new code.

Reply:
I had `Version: 0.66.0` installed ... I should probably update ...